dist/common/scripts/scylla_util.py

```diff
@@ -381,6 +381,8 @@ def _non_root_nvmes(self):
             raise Exception("found more than one disk mounted at root'".format(root_dev_candidates))
 
         root_dev = root_dev_candidates[0].device
+        if root_dev == '/dev/root':
+            root_dev = run('findmnt -n -o SOURCE /', shell=True, check=True, capture_output=True, encoding='utf-8').stdout.strip()
```

Comment:
A different way to do this is to read the inode of the filename, and look at /sys/dev/block/{major}:{minor}. That works for anything, not just /root. But we can start with your approach.

However, I need a Fixes # since we may need to backport this.

Reply:
Added issue number.